db.go

```diff
@@ -0,0 +1,29 @@
+package db
+
+import (
+	"database/sql"
+
+	_ "github.com/lib/pq"
+)
+
+// A DB is responsible for all interactions with postgres
+type DB struct {
```

Comment:
I don't think this file or struct should be in here, should probably be something like `sqlmigrate.Migratione(db *sql.DB, ______ args for migrations from bindata ____)`